ui/src/data/configuration.ts

```diff
@@ -13,6 +11,9 @@ export enum SortDirection {
   Desc = "DESC",
 }
 
+export const ROLLUP_COUNT_ATTRIBUTE = "t_rollup_count";
+export const VALUE_SUFFIX = "_t_value";
```

Comment:
Just to make sure I'm following your PR description, the UI is basically mapping all selected information to a column, and these are the hardcoded name/suffix of the relationship field column and the display columns for value-display attributes. Is that correct?

Reply:
Yes. All of the attributes for an instances are stored in map and each column pulls out its value by key. I could instead have them configured with something like `"rollupCount": true` instead of `"column": "t_rollup_count"` but in a list in a configuration file those are basically the same and the former is maybe a bit worse because it breaks the consistency of the items.

There's no `_display` suffix because the display property is used by default if it exists. In most cases that's the desired behavior but there can be exceptions. "Vocab" is the only place `_value` is used so far because the vocabulary descriptions are quite verbose and nobody actually uses them.